tests/core/test_libsnappy_compat.py

```diff
@@ -0,0 +1,22 @@
+from hypothesis import given, settings, strategies as st
+
+from py_snappy import compress, decompress
+from snappy import compress as libsnappy_compress, decompress as libsnappy_decompress
+
+MEGABYTE = 1000000
+
+
+@given(value=st.binary(min_size=1, max_size=2 * MEGABYTE))
```

Comment:
what happens with compressing `b''`? Is it also just the empty byte-string? Or is it invalid according to snappy?

Reply:
Not sure, I'll add explicit tests for `b''` as both an compressed value and a value needing to be compressed.